Core/src/Material/SurfaceMaterialMapper.cpp

```diff
@@ -252,7 +252,7 @@ void Acts::SurfaceMaterialMapper::mapMaterialTrack(
       double distVol = (volIter->position - mTrack.first.first).norm();
       double distMat = (rmIter->position - mTrack.first.first).norm();
       // Material past the entry point to the current volume
-      if (distMat > distVol) {
+      if (distMat - distVol > s_epsilon) {
```

Comment:
Can this never be < 0 ? I.e. would it make sense to protect with `std::abs()` - or is this a given from the entry point?

Reply:
Since both distMat and distVol are norm they should be > 0 .
The idea is that we want to change the volume we want to compare to if the material hit is not in the volume and further away from the origin than the entry point. 
I replace the the comparison in case due to the rounding the first hit would appear outside the volume but after the entry point. Rethinking about this might not even be needed.

Reply:
Understood.